addons/website_forum/tests/test_performance.py

```diff
@@ -30,9 +30,9 @@ def test_perf_sql_forum_standard_data(self):
         number_of_queries = self._get_url_hot_query(self.forum_website_url, cache=False)
         self.assertLessEqual(number_of_queries, 28)
         number_of_queries = self._get_url_hot_query(self.post.website_url)
-        self.assertEqual(number_of_queries, 19)
+        self.assertEqual(number_of_queries, 20)
```

Comment:
I'd be interested to know what is causing this extra request / which request is it
Don't spend too much time on it, or maybe just do it the "dumb" way by commenting lines to figure which one is causing it.

From what I expected, I thought that there would be no extra query since the page already display all the info we add in the schema js thing, no?

You can probably also just tweak `_get_url_hot_query()` to play with `table_count` to see which table got the extra query